flocker/dockerplugin/__init__.py

```diff
@@ -5,3 +5,13 @@
 
 This may eventually be a standalone package.
 """
+
+from twisted.internet.address import UNIXAddress
+
+# Many places in both twisted.web and Klein are unhappy with listening on
+# Unix socket, e.g.  https://twistedmatrix.com/trac/ticket/5406.  The
+# Docker plugin needs to listen using a Unix socket, so "fix" that by
+# pretending we have a port number and host. Yes, I feel guilty.
+UNIXAddress.port = 0
+UNIXAddress.host = b"127.0.0.1"
+del UNIXAddress
```

Comment:
I'm not sure why you think this won't happen in an actual Flocker process. This is only necessary for Docker plugin (previously it run in the docker plugin script's `main()` method) and anything involving docker plugin (test or real use) will import this package.